From 27b6c2739079fc0ec791ddead7fa5905e6a397aa Mon Sep 17 00:00:00 2001
From: Muhammad Falak R Wani <falakreyaz@gmail.com>
Date: Fri, 3 Mar 2023 16:54:18 +0530
Subject: [PATCH] autoload: fugitive: modify 'cc' to commit and sign-off

Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>
---
 autoload/fugitive.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoload/fugitive.vim b/autoload/fugitive.vim
index d29b953..0efcc9e 100644
--- a/autoload/fugitive.vim
+++ b/autoload/fugitive.vim
@@ -7658,7 +7658,7 @@ function! s:MapGitOps(is_ftplugin) abort
   exe s:Map('n', 'cv<Space>', ':tab Git commit -v<Space>', '', ft)
   exe s:Map('n', 'cv<CR>', ':tab Git commit -v<CR>', '', ft)
   exe s:Map('n', 'ca', ':<C-U>Git commit --amend<CR>', '<silent>', ft)
-  exe s:Map('n', 'cc', ':<C-U>Git commit<CR>', '<silent>', ft)
+  exe s:Map('n', 'cc', ':<C-U>Git commit -s<CR>', '<silent>', ft)
   exe s:Map('n', 'ce', ':<C-U>Git commit --amend --no-edit<CR>', '<silent>', ft)
   exe s:Map('n', 'cw', ':<C-U>Git commit --amend --only<CR>', '<silent>', ft)
   exe s:Map('n', 'cva', ':<C-U>tab Git commit -v --amend<CR>', '<silent>', ft)
-- 
2.39.2

